scripts/tmux-extrakto.sh

```diff
@@ -112,6 +120,11 @@ function capture() {
         capture
       fi
       ;;
+
+    ctrl-e)
+      tmux send-keys -t ! "$editor -- $text"
+      tmux send-keys -t ! 'C-m'
```

Comment:
You can do this in one command:
```
      tmux send-keys -t ! "$editor -- $text" 'C-m'
```

Reply:
Oh nice, didn't know that :slightly_smiling_face: